bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix: added missing publicly visible kafka params in orderby query

## Description
fix: added missing publicly visible kafka params in orderby query

## Verification Steps
1. run `make run/docs` and confirm that the openapi spec for kafkas get lists allowed orderby params
2. confirm that ci tests pass

## Checklist (Definition of Done)
